stko/optimizers/xtb.py

```diff
@@ -516,6 +518,422 @@ def optimize(self, mol):
         return mol
 
 
+class XTBCREST(Optimizer):
+    """
+    Uses GFN-n [1]_ to run CREST [2]_ on molecules.
+
+    Notes
+    -----
+    Requires version > 6.2 of xtb.
+
+    When running :meth:`optimize`, this calculator changes the
+    present working directory with :func:`os.chdir`. The original
+    working directory will be restored even if an error is raised, so
+    unless multi-threading is being used this implementation detail
+    should not matter.
+
+    If multi-threading is being used an error could occur if two
+    different threads need to know about the current working directory
+    as :class:`.XTBCREST` can change it from under them.
+
+    Note that this does not have any impact on multi-processing,
+    which should always be safe.
+
+    *Restrictions to iMTD-GC Algorithm*
+    Z-matrix sorting is forced to be off because :class:`stk.Molecules`
+    cannot have their ordering changed by an external program at this
+    stage. This may impact the effectiveness of the algorithm.
```

Comment:
How will this impact the effectiveness of the algorithm? Slightly unclear. Can it still be used? In what cases should the user be worried?

Reply:
The impact of this option is not clear to me. Z-matrix sorting is used to improve the conformed exploration (I believe), but I do not know how much of an impact it may have.